PointerStar/Client/Pages/Room.razor

```diff
@@ -4,11 +4,29 @@
 
 @inherits ComponentBase<RoomViewModel>
 
+@inject NavigationManager Navigation;
+    
 @page "/room/{RoomId}"
 
 <PageTitle>Pointer*</PageTitle>
 
-<h2>Room @ViewModel.RoomState?.RoomId</h2>
+<h2>Room @ViewModel.RoomState?.RoomId
```

Comment:
@Keboo  is this what you had in mind for separating the ui handling from logic ?